python/cudf/cudf/tests/test_json.py

```diff
@@ -1453,3 +1453,12 @@ def test_chunked_json_reader():
     with cudf.option_context("io.json.low_memory", True):
         gdf = cudf.read_json(buf, lines=True)
     assert_eq(df, gdf)
+
+
+@pytest.mark.parametrize("compression", ["gzip", None])
```

Comment:
There's a `compression_params` you could use

```suggestion
@pytest.mark.parametrize("compression", compression_params)
```